src/main/java/net/dv8tion/jda/api/entities/ThreadChannel.java

```diff
@@ -183,88 +296,472 @@ default ThreadMember getThreadMemberById(String id)
     @CheckReturnValue
     RestAction<ThreadMember> retrieveThreadMemberById(long id);
 
+    /**
+     * Retrieves the {@link ThreadMember ThreadMembers} of this thread.
+     *
+     * This requires the {@link net.dv8tion.jda.api.requests.GatewayIntent#GUILD_MEMBERS} intent to be enabled.
+     *
+     * @return a RestAction that resolves into a List of {@link ThreadMember ThreadMembers} of this thread.
+     */
+    //TODO-v5: docs - documentation depends on implementation cleanup.
```

Comment:
I don't understand this TODO

Reply:
The implementation doesn't work: see #1991